scrapper.py

```diff
@@ -26,38 +36,133 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
```

Comment:
this function *extracts* URLS, verifies them, but no cut is performed here

Reply:
Could you explain please what exactly should I do here?